### DISCUSSION TITLE
insertRow() is displaying syntax error

### DISCUSSION THREAD

**Discussion by mozexty** [serverpod/serverpod#2875] [Q&A] [answered]
I just created a new Serverpod 1.2.5 project. In the hello method of example_endpoint.dart, I tried this 
`await session.db.insertRow(Example(name: name, data: 1));`
Then it said _Couldn't infer type parameter T_

Then I tried this
`var example = await Example.db.insertRow(session, Example(name: name, data: 1));`
I got another error : _the getter db isn't defined for the type Example._

How to save a new Example entity ? Am I missing something ?

**Comment by hampuslavin** [comment, marked as answer]
Hi @mozexty! 👋 
The `Example` model that exists in new Serverpod project does not a have a database table specified in its model file (`lib/src/models/example.spy.yaml`). If you'd like to store it in the database, add the `table` key to the yaml file:
```yaml
table: examples
```
Then you have to create and run migrations. See these guides for complete instructions on this: 
- https://docs.serverpod.dev/next/get-started#working-with-the-database
- https://docs.serverpod.dev/next/concepts/database/models


If you haven't already, I would also recommend glancing through the entire Database section of our docs which covers alla aspects of the ORM: https://docs.serverpod.dev/next/concepts/database/connection
